src/hiccup/compiler.clj

```diff
@@ -49,10 +66,17 @@
   (or content
       (and (html-mode?) (not (void-tags tag)))))
 
+(defn class-list [value]
```

Comment:
This function doesn't seem relevant to the PR either? Could you explain its purpose?